src/test/java/com/commercetools/sync/inventory/InventorySyncTest.java

```diff
@@ -0,0 +1,214 @@
+
+    private final static String REF_1 = "111";
+    private final static String REF_2 = "222";
+    private final static String REF_3 = "333";
+
+    private static final String CTP_KEY = "testKey";
+    private static final String CTP_ID = "testId";
+    private static final String CTP_SECRET = "testSecret";
+
+    private final Long QUANTITY_1 = 10l;
+    private final Long QUANTITY_2 = 30l;
+
+    private final Integer RESTOCKABLE_1 = 10;
+    private final Integer RESTOCKABLE_2 = 10;
+
+    private final ZonedDateTime DATE_1 = ZonedDateTime.of(2017, 4, 1, 10, 0, 0, 0, ZoneId.of("UTC"));
+    private final ZonedDateTime DATE_2 = ZonedDateTime.of(2017, 5, 1, 20, 0, 0, 0, ZoneId.of("UTC"));
+
+    private List<InventoryEntryDraft> drafts;
+    private List<InventoryEntry> inventoriesDB;
+    private List<Channel> channelsDB;
+
+    {
+        Channel channel1 = mockChannel(REF_1, KEY_1);
+        Channel channel2 = mockChannel(REF_2, KEY_2);
+        Channel channel3 = mockChannel(REF_3, KEY_3);
+
+        Reference<Channel> reference1 = Channel.referenceOfId(REF_1).filled(channel1);
+
+        channelsDB = asList(channel1, channel2);
+        inventoriesDB = asList(
+                InventoryEntryMock.of(SKU_1, QUANTITY_1, RESTOCKABLE_1, DATE_1).build(),
+                InventoryEntryMock.of(SKU_1, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_1, KEY_1).build(),
+                InventoryEntryMock.of(SKU_1, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_2, KEY_2).build(),
+                InventoryEntryMock.of(SKU_2, QUANTITY_1, RESTOCKABLE_1, DATE_1).build(),
+                InventoryEntryMock.of(SKU_2, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_1, KEY_1).build(),
+                InventoryEntryMock.of(SKU_2, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_2, KEY_2).build()
+        );
+
+        drafts = asList(
+                InventoryEntryDraft.of(SKU_1, QUANTITY_1, DATE_1, RESTOCKABLE_1,null),
+                InventoryEntryDraft.of(SKU_1, QUANTITY_1, DATE_1, RESTOCKABLE_1, reference1),
+                InventoryEntryDraft.of(SKU_1, QUANTITY_1, DATE_1, RESTOCKABLE_1, Channel.referenceOfId(KEY_2)),
+                InventoryEntryDraft.of(SKU_2, QUANTITY_2, DATE_2, RESTOCKABLE_2,null),
+                InventoryEntryDraft.of(SKU_2, QUANTITY_2, DATE_2, RESTOCKABLE_2, reference1),
+                InventoryEntryDraft.of(SKU_2, QUANTITY_2, DATE_2, RESTOCKABLE_2, Channel.referenceOfId(KEY_2)),
+                InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1,null),
+                InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1, reference1),
+                InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1, Channel.referenceOfId(KEY_2))
+        );
+    }
+
+
+    @Test
+    public void syncDrafts_ShouldEndWithoutExceptions() {
+        getInventorySyncer(30, 1, false).syncDrafts(drafts);
```

Comment:
what is use to have a separate test case "no exception"? If exception is not expected just do a normal test with asserts.

(same in `syncDrafts_WithParallelOption_ShouldEndWithoutExceptions()` below)